devtools/platform-descriptor-json/src/main/resources/codestarts/quarkus/core/buildtool/gradle/codestart.yml

```diff
@@ -22,7 +24,8 @@ language:
           package-uberjar: gradle quarkusBuild --uber-jar
           package-native: gradle build -Dquarkus.package.type=native
           package-native-container: gradle build -Dquarkus.package.type=native -Dquarkus.native.container-build=true
-          build-ci: gradle build
+          build-ci-cmd: gradle
```

Comment:
You need to also add `build-ci-cmd` and `build-ci-args` to the `devtools/platform-descriptor-json/src/main/resources/codestarts/quarkus/core/buildtool/gradle-kotlin-dsl/codestart.yml` codestart